### DISCUSSION TITLE
How to use "/apis/plugins/helm/packages/v1alpha1/installedpackages"

### DISCUSSION THREAD

**Discussion by xingyunyang01** [kubeapps/kubeapps#5787] [Q&A] [answered]
Take the installation of airflow as an example, I don't know how to fill in the body shown below and what the fields mean, can you provide me with an example?

{
  "availablePackageRef": {
    "context": {
      "cluster": "string",
      "namespace": "string"
    },
    "identifier": "string",
    "plugin": {
      "name": "kapp_controller.packages",
      "version": "v1alpha1"
    }
  },
  "targetContext": {
    "cluster": "string",
    "namespace": "string"
  },
  "name": "string",
  "pkgVersionReference": {
    "version": "string"
  },
  "values": "string",
  "reconciliationOptions": {
    "interval": "string",
    "suspend": true,
    "serviceAccountName": "string"
  }
}

**Comment by absoludity** [comment, marked as answer]
Hi @xingyunyang01 . I'm assuming you got that json format for the request from the inbuilt API documentation portal?

![kubeapps-api-doc](https://user-images.githubusercontent.com/497518/206951571-519c7ec1-e7de-4a3a-b880-7037ec455dca.png)

If so, you can scroll further down to see the documentation for the different message types. In this case, it's the `CreateInstalledPackageRequest`:

![create-installed-package-request](https://user-images.githubusercontent.com/497518/206951790-fd68da59-74f0-4c9b-a178-34685ac4def9.png)

That should provide you with more information about each of the fields and what is required. Note that this API is still alpha and may change.

**Reply by xingyunyang01** [reply]
Thanks，I have followed the documentation to call the API successfully
